src/containers/SideNav/SideNav.js

```diff
@@ -97,70 +97,132 @@ class SideNav extends Component {
+              isActive={
+                !!matchPath(location.pathname, {
+                  path: this.getPath(urls.tasks.all())
+                })
+              }
               to={this.getPath(urls.tasks.all())}
             >
               Tasks
             </SideNavMenuItem>
-            <SideNavMenuItem element={NavLink} to={urls.clusterTasks.all()}>
+            <SideNavMenuItem
+              element={NavLink}
+              isActive={
+                !!matchPath(location.pathname, {
+                  path: urls.clusterTasks.all()
+                })
+              }
+              to={urls.clusterTasks.all()}
+            >
               ClusterTasks
             </SideNavMenuItem>
             <SideNavMenuItem
               element={NavLink}
+              isActive={
+                !!matchPath(location.pathname, {
+                  path: this.getPath(urls.taskRuns.all())
+                })
+              }
               to={this.getPath(urls.taskRuns.all())}
             >
               TaskRuns
             </SideNavMenuItem>
             <SideNavMenuItem
               element={NavLink}
+              isActive={
+                !!matchPath(location.pathname, {
+                  path: this.getPath(urls.conditions.all())
+                })
+              }
               to={this.getPath(urls.conditions.all())}
             >
               Conditions
             </SideNavMenuItem>
             {this.props.isTriggersInstalled && (
-              <>
-                <SideNavMenuItem
-                  element={NavLink}
-                  to={this.getPath(urls.eventListeners.all())}
-                >
-                  EventListeners
-                </SideNavMenuItem>
-                <SideNavMenuItem
-                  element={NavLink}
-                  to={this.getPath(urls.triggerBindings.all())}
-                >
-                  TriggerBindings
-                </SideNavMenuItem>
-                <SideNavMenuItem
-                  element={NavLink}
-                  to={urls.clusterTriggerBindings.all()}
-                >
-                  ClusterTriggerBindings
-                </SideNavMenuItem>
-                <SideNavMenuItem
-                  element={NavLink}
-                  to={this.getPath(urls.triggerTemplates.all())}
-                >
-                  TriggerTemplates
-                </SideNavMenuItem>
-              </>
+              <SideNavMenuItem
+                element={NavLink}
+                isActive={
+                  !!matchPath(location.pathname, {
+                    path: this.getPath(urls.eventListeners.all())
+                  })
+                }
+                to={this.getPath(urls.eventListeners.all())}
+              >
+                EventListeners
+              </SideNavMenuItem>
+            )}
+            {this.props.isTriggersInstalled && (
+              <SideNavMenuItem
+                element={NavLink}
+                isActive={
```

Comment:
Alot of duplication in these `isActive` parameters, its not big but if you want to move them to a function that takes the url. Ill leave it up to you, how it is is fine too

Reply:
Yeah I thought about that when I discovered the [nesting issue](https://github.com/tektoncd/dashboard/pull/1855#discussion_r535180532) but figured it didn't really bring much benefit so left it. I may have a solution / workaround to the nesting issue for a follow up PR, if that doesn't pan out I'll fix this up then if that's alright.